flexget/plugins/plugin_rtorrent.py

```diff
@@ -542,6 +542,7 @@ def add_entry(self, client, entry, options, start=True, mkdir=False):
 
         if entry['url'].startswith('magnet:'):
             torrent_raw = 'd10:magnet-uri%d:%se' % (len(entry['url']), entry['url'])
+            torrent_raw = str.encode(torrent_raw)
```

Comment:
`torrent_raw.encode('ascii')` or `bytes(torrent_raw, 'ascii')` would be the more canonical way to write that.